sdlib/d/gc/page.d

```diff
@@ -674,6 +674,32 @@ private:
 		}
 	}
 
+	static void finalizeSlabNimble(size_t nimbleBits, int sizeClass,
+	                               Extent* extent, size_t indexOffset) {
```

Comment:
You only need the address from the extent, so pass that in.

Reply:
I was trying to avoid accessing lots of extra stuff if no finalizers need to run (i.e. nimbleBits is 0). Aren't we trying to avoid touching things in the hot loops?